src/cclib/writer/xyzwriter.py

```diff
@@ -36,7 +36,6 @@ def __init__(self, ccdata, splitfiles=False,
         self.do_lastgeom = lastgeom
         self.do_allgeom = allgeom
 
-        self.generate_repr()
```

Comment:
It's called in the parent script [ccwrite.py](https://github.com/cclib/cclib/blob/master/src/cclib/writer/ccwrite.py#L71) once the filewriter has been instantiated.